modules/imgproc/include/opencv2/imgproc.hpp

```diff
@@ -1886,7 +1886,7 @@ CV_EXPORTS_W void Laplacian( InputArray src, OutputArray dst, int ddepth,
 //! @addtogroup imgproc_feature
 //! @{
 
-/** @example samples/cpp/edge.cpp
+/** @example samples/dnn/edge_detection.cpp
```

Comment:
We still need simplified API usage example.
Code with 200+ lines are just not suitable for initial leaning steps.

We need 2 kind of "samples":
- simplified API usage example (e.g. move to part of documentation - tutorial_code)
- large "task solution" "demos"

Reply:
edge.cpp has been moved to samples/cpp/snippets directory as it has been done for other basic cpp samples